README.md

```diff
@@ -1 +1,73 @@
 # synthetic-scripting
+
```

Comment:
Before I start reviewing the code, its hard to tell what this integration is used for. Can you add a short description to the readme of what this integration actually does?